designs/administrator/olm/sort-by-recently-updated/sort-by-recently-updated.md

```diff
@@ -0,0 +1,48 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Sort by Provider update on OperatorHub
+
+When viewing the list of Operators available for install on OperatorHub, the users are interested in which Operators were updated most recently. OperatorHub should support sorting by the most recently updated Operator.
+
+## Sorting on OperatorHub
+
+The Sort by menu and View icons allow the user to now further customize their sorting experience on Operator Hub.
+
+![Sort-by-on-OperatorHub](img/1-filter-on-operator-hub.jpg)
+
+## User selects “Provider update” on menu
+
+When the user selects “Provider update” from the Sort by menu, the information about when this operator was last updated appears in the bottom left of the card.
+
+![Sort-by-on-OperatorHub2](img/2-filter-on-operator-hub.png)
+
+## “Updated by provider” appears on card under Operator Title and Provider
+
+Now the Sort by dropdown has been updated with the Provider update selection, and the timestamp appears under Operator Title and Provider.
+
+![Sort-by-on-OperatorHub3](img/2.5-filter-on-operator-hub.jpg)
+
+## “Updated by provider” timestamp appears on bottom left of Operator description
+
+Once clicking into the 3Scale Operator the user can view the Updated by provider time stamp in replacement of the Created at field which should be removed.
+
+![Updated-provider-operator-details](img/3-filter-on-operator-hub.png)
+
+## User selects Capability level
+
+When the user selects Capability level, the Operators with the highest level of capability are listed first from 5 to 1 in descending order.
```

Comment:
Ah, now I see why it might be difficult to always show this information :) I guess eventually there will be information overload if we add it all, haha. Would badges be something to explore here to help the data stand out?

Reply:
Yes, that's right. We feel that the information the user sorts by should be visually differentiated by other information that appears in badges on a card. Megan is doing work on this story, and will be sharing that soon. She's collaborated on these efforts and recommendations that Michael Celedonia helped us make from the visual design perspective.

Reply:
Great thanks for following up!